src/vtkIECTransformLogic.h

```diff
@@ -121,12 +131,62 @@ class VTK_IEC_TRANSFORM_LOGIC_EXPORT vtkIECTransformLogic : public vtkObject
+  void UpdateTableTopEccentricRotationToPatientSupportRotationTransform(double TableTopEccentricRotationAngleDeg, double ey = 0);
+  /// @brief Update TableTopToTableTopEccentricRotation transform based on Table Top displacement(in X,Y,Z), Table Top pitch rotation(around X axis), and Table Top roll rotation(around Y axis) parameters
+  /// Starting from the table top eccentric frame displacement of the origin is performed followed by table top pitch rotation around the X-axis and then table top roll rotation around the Y-axis
+  /// @param tx displacement of the table top frame origin along the X-axis
+  /// @param ty displacement of the table top frame origin along the Y-axis
+  /// @param tz displacement of the table top frame origin along the Z-axis
+  /// @param tableTopPitchAngleDeg the rotation of the table top eccentric frame counter clockwise around the X-axis
+  /// @param tableTopRollAngleDeg the rotation of the table top eccentric frame counter clockwise around the Y-axis
+  void UpdateTableTopToTableTopEccentricRotationTransform(double tx, double ty, double tz, double tableTopPitchAngleDeg = 0, double tableTopRollAngleDeg = 0);
+  /// @brief Update PatientToTableTop transform based on patient displacement(in X,Y,Z) and patient rotation around X(Psi), Y(Phi) and Z(Theta) parameters
+  /// Starting from the table top frame displacement of the origin is performed followed by Psi rotation around the X-axis, Phi rotation around the Y-axis, and then Theta rotation around the Z-axis respectively
+  /// @param px displacement of the patient frame origin along the X-axis
+  /// @param py displacement of the patient frame origin along the Y-axis
+  /// @param pz displacement of the patient frame origin along the Z-axis
+  /// @param patientPsiAngleDeg the rotation of the table top eccentric frame counter clockwise around the X-axis
+  /// @param patientPhiAngleDeg the rotation of the table top eccentric frame counter clockwise around the Y-axis
+  /// @param patientThetaAngleDeg the rotation of the table top eccentric frame counter clockwise around the Z-axis
+  void UpdatePatientToTableTopTransform(double px, double py, double pz, double patientPsiAngleDeg = 0, double patientPhiAngleDeg = 0, double patientThetaAngleDeg = 0);
+  /// @brief Update PatientImageRegularGrid transform based on the parameters
+  /// The transformation matrix from DICOM(patient frame in LPS format) to patient image regular grid frame of the following format is defined
+  ///     directionCosineXx*columnPixelSpacing, directionCosineYx*rowPixelSpacing, directionCosineZx*sliceDistance, sx
+  /// m = directionCosineXy*columnPixelSpacing, directionCosineYy*rowPixelSpacing, directionCosineZy*sliceDistance, sy
+  ///     directionCosineXz*columnPixelSpacing, directionCosineYz*rowPixelSpacing, directionCosineZz*sliceDistance, sz
+  ///                                        0,                                 0,                               0,  1
```

Comment:
```suggestion
  /// The transformation matrix from DICOM(patient frame in LPS format) to patient image regular grid frame of the following format is defined, see Annex B of IEC61217:2011 p. XXXX? and https:...
```
No need to put the formula here in the comments.